gensim/test/test_ldamallet_wrapper.py

```diff
@@ -95,6 +95,18 @@ def testSparseTransform(self):
                             (i, sorted(vec), sorted(expected)))
         self.assertTrue(passed)
 
+    def testMallet2Model(self):
+        if not self.mallet_path:
+            return
+        passed = False
+        tm1 = ldamallet.LdaMallet(self.mallet_path, corpus=corpus, num_topics=2, id2word=dictionary)
+        tm2 = ldamallet.malletmodel2ldamodel(tm1)
+        for document in corpus:
+            self.assertAlmostEqual(tm1[document][0][1], tm2[document][0][1], places=1)
+            self.assertAlmostEqual(tm1[document][1][1], tm2[document][1][1])
```

Comment:
@piskvorky I'm not too sure about this test. Values sometimes are differing at the first place too.